prepare_data.py

```diff
@@ -148,7 +148,7 @@ def main():
     if data_args.model_type == 't5':
         tokenizer = T5Tokenizer.from_pretrained("t5-base")
     else:
-        tokenizer = T5Tokenizer.from_pretrained("bart-base")
+        tokenizer = T5Tokenizer.from_pretrained("facebook/bart-base")
```

Comment:
```suggestion
        tokenizer = BartTokenizer.from_pretrained("facebook/bart-base")
```

Aah, just found another issue, bart tokenizer should be loaded using `BartTokenizer` class